sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/MethodsShouldNotHaveTooManyLines_CustomValues.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+
+namespace Tests.Diagnostics
+{
+    class Program
```

Comment:
Other cases:
* multi-and single line comments
* properties (to show they are not reported on)
* method that have tokens on the same line as the opening/closing brace.